2006-04-04  Roland McGrath  <roland@redhat.com>

	[BZ #2508]
	* nscd/Makefile ($(objpfx)nscd_stat.o: sysincludes): Move setting
	inside [$(have-selinux) = yes].
	($(objpfx)selinux.o: sysincludes): Likewise.

Index: nscd/Makefile
===================================================================
RCS file: /cvs/glibc/libc/nscd/Makefile,v
retrieving revision 1.48
retrieving revision 1.49
diff -u -p -r1.48 -r1.49
--- nscd/Makefile	16 Dec 2005 04:06:11 -0000	1.48
+++ nscd/Makefile	4 Apr 2006 07:32:46 -0000	1.49
@@ -58,9 +59,6 @@ endif
 
 nscd-modules += selinux
 selinux-LIBS := -lselinux $(libaudit)
-endif
-
-LDLIBS-nscd = $(selinux-LIBS)
 
 # The configure.in check for libselinux and its headers did not use
 # $SYSINCLUDES.  The directory specified by --with-headers usually
@@ -69,6 +67,9 @@ LDLIBS-nscd = $(selinux-LIBS)
 # system headers will be ok for this file.
 $(objpfx)nscd_stat.o: sysincludes = # nothing
 $(objpfx)selinux.o: sysincludes = # nothing
+endif
+
+LDLIBS-nscd = $(selinux-LIBS)
 
 distribute := nscd.h nscd-client.h dbg_log.h \
 	      $(addsuffix .c, $(filter-out xmalloc,$(all-nscd-modules))) \
